src/App.tsx

```diff
@@ -43,6 +43,8 @@ function AppContainer() {
 
   const [cancelUpload, setCancelUpload] = useState(false);
   const [isCreatingSet, setIsCreatingSet] = useState(false);
+  const [isSorting, setIsSorting] = useState(false);
+  const [sorting, setSorting] = useState<SortingState>([]);
```

Comment:
I'm a little confused why we need both of these.

I would think you could have a variable like

`const isSorting = sorting.length > 0 && (sorting.length === 1 && sorting[0].id !== "row_order");`

or something like that. that second condition may need to be updated to only be desc? can't really tell

Then you wouldn't need to manually track it. I think this would also fix the bug of where you unsort by clicking the table header a 3rd time, `isSorting` is staying true and preventing you from reordering.